mkdocs/tests/config/config_options_tests.py

```diff
@@ -494,6 +494,50 @@ def test_dir_is_config_dir_fails(self):
         self.assertEqual(len(warns), 0)
 
 
+class ListOfPathsTest(unittest.TestCase):
+
+    def test_valid_path(self):
+        paths = [os.path.dirname(__file__)]
+        option = config_options.ListOfPaths()
+        option.validate(paths)
+
+    def test_missing_path(self):
+        paths = [os.path.join("doesnt", "exist", "i", "hope")]
```

Comment:
```suggestion
        paths = [os.path.join("does", "not", "exist", "i", "hope")]
```
the last linter failure 😀😖